relnotes/snaketopascal.feature.md

```diff
@@ -0,0 +1,5 @@
+### snakeToPascal
```

Comment:
This should be a catchy title/description of the change.

Reply:
Ha, I've never heard of Pascal case before. Looks like the correct term though :+1:

Reply:
Cause it's actually called UpperCamelCase. Pascal just happened to use it and now ppl call it this way. So now PascalCase is supposed to be a subset of CamelCase OR plainly UpperCamelCase.

Reply:
Right, UpperCamelCase is the term I've heard before. As long as it's a standard term, I think either is fine though.